Gemfile

```diff
@@ -1,6 +1,7 @@
 source 'https://rubygems.org'
 
 gem 'guard-rspec', require: false
-gem 'spree', github: 'spree/spree', branch: 'master'
+gem 'spree', github: 'spree/spree', branch: '3-0-stable'
```

Comment:
You're right in that it should be `master`. This was a proposal initially which is still being worked on.

Also, the `3-0-stable` branch is not nonexistent: https://github.com/spree/spree/tree/3-0-stable